demo/src/main.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable @typescript-eslint/quotes */
+/* eslint-disable no-undef */
+import { CommandBridge } from 'jupyter-iframe-commands-host';
+import Swal from 'sweetalert2';
```

Comment:
Also wondering if we could even use the native `<dialog>` directly here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog